lib/plugins/aws/package/compile/events/apiGateway/lib/restApi.js

```diff
@@ -63,6 +79,15 @@ module.exports = {
           Policy: policy,
         }
       );
+    } else {
+      _.merge(
+        this.serverless.service.provider.compiledCloudFormationTemplate.Resources[
```

Comment:
So if I understand right, the workaround is to ensure policy in all cases, and when user didn't provide any, setup one that has no restrictions.

If it's the case, it'll be good to add a comment to explain why we're doing that.

Reply:
@medikoo You are correct.  I will add a comment to explain.